marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/artifacts/mappings.sjs

```diff
@@ -83,6 +83,17 @@ function getArtifactSettingNode(collectionName, artifactName, artifactVersion) {
   return fn.head(results);
 }
 
+function defaultArtifactSettings(artifactName) {
+    return {
+        artifactName,
+        collections: ['default-mapping'],
```

Comment:
@ryanjdew wondering if this should be default-mapping or entity-services-mapping?

Reply:
default-mapping is the collection set by entity-services-mapping step definition.